setup.py

```diff
@@ -5,14 +5,16 @@
     include_package_data=True,
     long_description_content_type="text/markdown",
     install_requires=[
-        "dbus-python==1.2.16",
-        "Flask==1.1.2",
-        "Flask-SocketIO==5.0.1",
-        "eventlet==0.31.0",
+        "Flask==2.1.3",
```

Comment:
Could we potentially bump this to the latest version for future-proofing? I'm seeing 2.3.2 as the latest version. This may also tie into your pinning of the the other dependencies, if updated.

Reply:
> Could we potentially bump this to the latest version for future-proofing? I'm seeing 2.3.2 as the latest version. This may also tie into your pinning of the the other dependencies, if updated.

Well I hand-picked those versions because they were the only ones that worked with the script, probably because that the script is outdated for the newest version of Flask, but if you plan on updating it, it should be alright with `Flask==2.3.2`.

Reply:
Alright, thanks for the heads up on that. Merging on this version and tackling the tech debt in the webapp script, sounds like a good plan.